code/modules/industrial_lift/crossing_signal.dm

```diff
@@ -29,10 +29,10 @@
 	var/tram_id = MAIN_STATION_TRAM
 	/// Weakref to the tram piece we control
 	var/datum/weakref/tram_ref
-	/// Proximity threshold for amber warning (slow people may be in danger)
+	/// Proximity threshold for amber warning (slow people may be in danger) This is specific to Tramstation and may need to be adjusted if the map changes in the distance between tram stops.
 	var/amber_distance_threshold = 60
-	/// Proximity threshold for red warning (running people will likely not be able to cross)
-	var/red_distance_threshold = 35
+	/// Proximity threshold for red warning (running people will likely not be able to cross) This is specific to Tramstation and may need to be adjusted if the map changes in the distance between tram stops.
```

Comment:
I want more documentation on *how* you arrived to this number, so that when Tramstation is changed, we at least have a clue on how this number was reached.

Reply:
I adjusted it 1 at a time until the signals were showing properly.

Reply:
Document this as a comment, and describe what "showing properly" means